presentation/src/main/java/com/wednesday/template/presentation/weather/start/StartFragment.kt

```diff
@@ -1,26 +1,40 @@
 package com.wednesday.template.presentation.weather.start
 
+import android.util.Log
+import androidx.fragment.app.viewModels
 import com.wednesday.template.presentation.base.effect.Effect
 import com.wednesday.template.presentation.base.fragment.BindingProvider
 import com.wednesday.template.presentation.base.fragment.MainFragment
 import com.wednesday.template.presentation.base.toolbar.ToolbarComponent
 import com.wednesday.template.resources.databinding.FragmentStartBinding
 
-class StartFragment : MainFragment<
+    class StartFragment : MainFragment<
     FragmentStartBinding,
     StartFragmentScreen,
     StartScreenState,
     StartScreenViewModel>() {
+    
+    override val toolbarComponent: ToolbarComponent = ToolbarComponent(this)
 
-    override val viewModel: StartScreenViewModel
-        get() = TODO("Not yet implemented")
-    override val bindingProvider: BindingProvider<FragmentStartBinding>
-        get() = TODO("Not yet implemented")
+    override val viewModel: StartScreenViewModel by navViewModel()
+    
+    override val bindingProvider: BindingProvider<FragmentStartBinding> = FragmentStartBinding::inflate
 
     override fun onEffect(effect: Effect) {
-        TODO("Not yet implemented")
+    }
+    
+    override fun onViewCreated(binding: FragmentStartBinding) {
+        super.onViewCreated(binding)
+        Log.d("*************","Calledddddddddddddddd")
+        initializeList(binding)
```

Comment:
Let's remove this. We will use the `ListComponent` and update it in `onState`

Reply:
Yeah I have removed it.